Windows10-swiss-knife/scripts/setup.ps1

```diff
@@ -0,0 +1,20 @@
+choco install -y googlechrome
+choco install -y visualstudio2019community
+choco install -y visualstudio2019-workload-nativedesktop
+choco install -y visualstudio2019-workload-vctools
+# Add mandatory component for Mimikatz compilation
+choco install -y visualstudio2017-workload-vctools --package-parameters "--add Microsoft.VisualStudio.Component.WinXP --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.WinXP --includeRecommended"
+choco install -y firefox
+choco install -y 7zip
+choco install -y notepadplusplus
+choco install -y winpcap
+choco install -y wireshark
+choco install -y processhacker
+choco install -y windbg -params '"/SymbolPath:c:\symbols"'
```

Comment:
I rather prefer setting the global variable `_NT_SYMBOL_PATH` to have the benefit of symbol caching on common tools.
I can do the changes if needed

Reply:
The environment variable is already sets to "SymbolPath" by default (thanks chocolatey !)